charts/harness-delegate/templates/harness-delegate.yml

```diff
@@ -99,7 +99,7 @@ spec:
             apt-get update
             # Install Git with auto approval
             yes | apt-get install git
-            yes | apt-get install google-cloud-cli jq wget
```

Comment:
a minor thing but we can combine two apt commands  ;-) itching my brain, you know